modules/oxide/src/less/theme/components/statusbar/statusbar.less

```diff
@@ -81,8 +81,8 @@
   .tox-statusbar__branding {
     svg {
       fill: currentColor;
-      height: 1.3em;
-      vertical-align: -.3em;
+      height: 1.4em;
```

Comment:
One thing I just thought of... since this will be shared between 5.x and 6.x skins, this might need to be a variable and adjusted accordingly. I don't know enough about how that's all setup though, but just flagging it as a possible issue.

Reply:
Actually, that's only an issue if we're showing the text for 5.x. I don't believe we are so this probably isn't an issue.

Reply:
I think the 6.x skin PR can worry about that after the logo has merged :)